content/docs/code-splitting.md

```diff
@@ -11,8 +11,13 @@ Most React apps will have their files "bundled" using tools like
 Bundling is the process of following imported files and merging them into a
 single file: a "bundle". This bundle can then be included on a webpage to load
 an entire app at once.
+Більшість React додатків мають власні розбиті за "бандлами" файли використовуючи такі інструменти, як
```

Comment:
Please remove translated lines from the file.

Reply:
@klymenkoo what do you think about `bundled` translation?

Reply:
fixed

Reply:
@ApolllonDev , btw, I saw translation like this at the Ukrainian front end websites such as https://codeguida.com, so I followed their translation :)